### PR TITLE
fixes and adds a method to torque layer

### PR DESCRIPTION
**Pull request by javisantana**
this is needed for a DI.js fix I'll push

cc @viddo

### CODE REVIEW

File: src/geo/leaflet/leaflet-torque-layer.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ var LeafletTorqueLayer = L.TorqueLayer.extend({
       this.provider.templateUrl = this.model.get('urls').tiles[0];
       // set meta
       _.extend(this.provider.options, this.model.get('meta'));
+      this.model.set(this.model.get('meta'));
```
Comment (viddo) [inline src/geo/leaflet/leaflet-torque-layer.js]: This is only needed for leaflet? Not for the gmaps one?

File: src/geo/map/torque-layer.js [hunk 2]
```diff
@@ -90,6 +90,15 @@ var TorqueLayer = MapLayer.extend({
   },
 
   fetchAttributes: function(layer, featureID, callback) {
+  },
+
+  // given a timestamp returns a step (float)
+  timeToStep: function(timestamp) {
```
Comment (viddo) [inline src/geo/map/torque-layer.js]: Isn't this [overriding the timeToStep method on the L.TorqueLayer](https://github.com/CartoDB/torque/search?utf8=%E2%9C%93&q=timeToStep)? Is that on purpose?

File: src/geo/leaflet/leaflet-torque-layer.js (hunk 1 above)
Comment (javisantana) [inline src/geo/leaflet/leaflet-torque-layer.js]: I guess both ones, will change

File: src/geo/map/torque-layer.js (hunk 2 above)
Comment (javisantana) [inline src/geo/map/torque-layer.js]: take into account this is the model not the view

**Comment by javisantana** [comment]
retest this please

**Comment by javisantana** [comment]
retest this please

**Comment by javisantana** [comment]
tired of CI not realizing about a new package, what a pile of shit

**Pull request merged**